### DISCUSSION TITLE
Lightning talks length?

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1159] [Suggested Changes]
I read somewhere the length for a lightning talk can be 15 minutes long

That seems... really long to me. At that point, isn't it just a regular talk...? 😅

My expectation is that lightning talks are 5 minutes long max. So I would suggest changing it to that

If the desire is to keep it at 15 minutes, I would suggest renaming it to something like a coffee talk ☕️

**Comment by meg-gutshall** [comment]
I'm pretty sure 15 minutes is what we capped it at last year, judging by the length of some of those videos. You do have a good point about the concept of a Lightning Talk. 

Maybe a 10 minute maximum would be better. I think we'll have to see how many speakers register over the weekend and decide from there.

**Reply by danieltott** [reply]
@BekahHW might have an opinion here but my take would be cap at 10 - some people will go over and stuff which is probably what happened in the past (I know I did 😂)

**Reply by ebanner** [reply]
I will take this opportunity to more fully express that my view is that a Lighting Talk is 2-5 minutes long and that a close to 10 minute talk has a different feel 

I think calling a 10 minute talk a Short Talk makes sense 

Lighting Talks to me have a very distinctive feel. Like the pace is actually intentionally sped up to cram stuff in. It's kind of a race against the clock. It's this little bite-sized thing. Like BAM! here it is and let's talk afterward

**Comment by BekahHW** [comment]
I believe we said 7 minutes, unless folks talked to me. And Dan always goes way over so we make him go last. On Apr 13, 2024, at 12:09 PM, Dan Ott ***@***.***> wrote:﻿
@BekahHW might have an opinion here but my take would be cap at 10 - some people will go over and stuff which is probably what happened in the past (I know I did 😂)
